roles/webots/directorrobocup.role

```diff
@@ -27,7 +33,6 @@ nuclear_role(
   skill::ScriptKick
   skill::GetUp
   skill::Look
-  skill::Dive
```

Comment:
How come removing the Dive? I've got a goalie branch that will use this.

Reply:
Not sure why this was removed, maybe when I merged another branch in.